docs/measure_scripts.md

```diff
@@ -0,0 +1,25 @@
+# Measure scripts
+
+FINESSE allows users to trigger a sequence of recordings with the mirror at different
+angles. The exact sequence, defined in terms of what the sequence of angles is and how
+many repeats to do, can be saved to a files, which we are calling "measure scripts".
+
+These files are written in [YAML](https://yaml.org/) format, with certain properties.
+Here is an example:
+
+```yaml
+measurements:
+  count: 10
+  sequence:
+  - angle: zenith
+    count: 1
+  - angle: 10.0
+    count: 3
+```
```

Comment:
Why the top level `measurement`? This can be written in a more simpler form as:

```suggestion
```yaml
count: 10
sequence:
- angle: zenith
  count: 1
- angle: 10.0
  count: 3
```
```

Reply:
Also, I would replace the top level `count` by `repeat` and the inner `count` by `measurements` (or `meas`), so their meaning is more explicit.

Reply:
In my head I had the idea that we might want to add another top-level key at some point. We probably should have a way of versioning the file format (with a `version` key or something) in case we decide to add features in future. And it sort of files like that info shouldn't be at the same level as things that are *actually* to do with how measurements will be recorded, e.g. `count` etc. What do you think?

I take your point about the other keys and I'll change those.

Reply:
Ok. Is there any hint that other features will be necessary in a measure script in the future? If not, I would not make the nesting deeper than needed - certainly not if the only other reason is to keep `version` - which I think it is a very appropriate field - separated from the others.

Reply:
Okey doke. These scripts are only ever going to be for describing the parameters to collect measurements, so I can't see there being any more top-level keys needed. Let's just drop the `measurements` one.

I suppose it would probably be useful to have a top-level `metadata` key with the creation date, program version etc. too as well as the file format version while we're at it.

Reply:
Having that top level metadata key with other stuff makes more sense to me than the measurement one, so go ahead with it if you want.